### PR TITLE
Font Change + Repo Reference Updates

### PR DESCRIPTION
**Pull request by leflambeur**

### CODE REVIEW

File: assets/scss/common/_fonts.scss [hunk 1]
```diff
@@ -69,3 +69,95 @@
...
+}
+/* open-sans-800italic - latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: italic;
+  font-weight: 800;
+  src: local(''),
+       url('fonts/vendor/open-sans/open-sans-v20-latin-800italic.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
+       url('fonts/vendor/open-sans/open-sans-v20-latin-800italic.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
```
Comment (JAG-UK) [inline assets/scss/common/_fonts.scss:163]: Do we not also want the bold face? Do you think browser approximated rendering will good enough?

Comment (leflambeur) [reply assets/scss/common/_fonts.scss:163]: I must have missed it in the download, let me double check

Comment (leflambeur) [reply assets/scss/common/_fonts.scss:163]: @JAG-UK They are there - 600-800 are bold

There are italic and non-italic versions

Ref: https://fonts.google.com/specimen/Open+Sans

Comment (JAG-UK) [reply assets/scss/common/_fonts.scss:163]: Ah, apologies. All good to go then

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**